Common/src/toolboxes/C1DInterpolation.cpp

```diff
@@ -148,7 +148,7 @@ vector<su2double> CorrectedInletValues(const vector<su2double> &Inlet_Interpolat
   /*---For turbulence variables columns---*/
   if (nVar_Turb == 1)
     Inlet_Values[nDim+5] = Inlet_Interpolated[5];
-  else if (nVar_Turb == 2)
+  if (nVar_Turb == 2)
```

Comment:
It was not by accident, nVar_Turb cannot be both 1 and 2, only one of those branches executes, if your intention is to execute both branches when nVar_Turb is 2, the code is wrong.

Reply:
@pcarruscag Thanks for the correction. I have changed the code, correctly with the else-if.

>   else if (nVar_Turb == 2){
    Inlet_Values[nDim+5] = Inlet_Interpolated[5];
    Inlet_Values[nDim+6] = Inlet_Interpolated[6];
  }